src/chrome/content/rules/LinuxFoundation.xml

```diff
@@ -40,100 +33,39 @@
 	<target host="lists.linux-foundation.org" />
```

Comment:
This PR is just bringing over changes from PR https://github.com/EFForg/https-everywhere/pull/6035. I don't want to get into expanding coverage beyond that.